File: Python/iniciante/1013maior.py
import math as m
linha = input().split(" ")
a, b, c = linha
a = int(a)
b = int(b)
c = int(c)
maior = (a + b + m.fabs(float(a) - float(b))) / 2
maior = (maior + c + m.fabs(float(maior) - float(c))) / 2
print("%.0lf eh o maior" % maior)

File: C/Iniciante/1013maior.c
#include <stdio.h>
#include <math.h>
int main() {
    int A,B,C;
    double maior;
    scanf("%i %i %i",&A,&B,&C);
    maior=(A+B+fabs(A-B))/2;
    maior=(maior+C+fabs(maior-C))/2;
    printf("%.0lf eh o maior\n",maior);
    return 0;
}